### ISSUE TITLE
update `jake` to remove `async` vulnerability

### ISSUE THREAD

**Issue by userquin** [mde/ejs#668]
`"jake": "^10.6.1"` is using `async 0.9.x`
```md
┌─────────────────────┬───────────────────────────────────────────────────┐
│ high                │ Prototype Pollution in async                      │
├─────────────────────┼───────────────────────────────────────────────────┤
│ Package             │ async                                             │
├─────────────────────┼───────────────────────────────────────────────────┤
│ Vulnerable versions │ <2.6.4                                            │
├─────────────────────┼───────────────────────────────────────────────────┤
│ Patched versions    │ >=2.6.4                                           │
├─────────────────────┼───────────────────────────────────────────────────┤
│ More info           │ https://github.com/advisories/GHSA-fwr7-v2mv-hh25 │
└─────────────────────┴───────────────────────────────────────────────────┘
```

**Comment by rouvenha** [comment]
Fix in #669 
Please merge and push a new version

**Comment by JackHowa** [comment]
@rouvenha @userquin this has been addressed and can be closed
